server/routes/users.js

```diff
@@ -1,55 +1,104 @@
 const router = require('express').Router();
```

Comment:
More useful responses from route endpoints.
Example of using protected route with router.delete('/', loginRequired, async (req, res) => {